mmdet/models/dense_heads/solo_head.py

```diff
@@ -0,0 +1,1173 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import mmcv
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn import ConvModule
+
+from mmdet.core import InstanceData, mask_matrix_nms, multi_apply
+from mmdet.core.utils import center_of_mass, generate_coordinate
+from mmdet.models.builder import HEADS, build_loss
+from .base_mask_head import BaseMaskHead
+
+
+@HEADS.register_module()
+class SOLOHead(BaseMaskHead):
+    """SOLO mask head used in  `<https://arxiv.org/abs/1912.04488>`_ .
```

Comment:
Beter to indicate the paper name rather than simply use a link, considering 
```
"""
`SOLO: Segmenting Objects by Locations <https://arxiv.org/abs/1912.04488>`_
```
Can check  https://mmdetection--5832.org.readthedocs.build/en/5832/ for the docstring format